Homework3

LeahLeah88 opened the pull request:
Домашнее задание №3, Ольга Лосева.

LeahLeah88 committed:
Add project files.

Add project files.

LeahLeah88 committed:
Add project files.

Add project files.

LeahLeah88 committed:
Delete HW-3.vcxproj.filters

Удаление файла HW-3.vcxproj.filters вне папки.

LeahLeah88 committed:
Add project files.

Add project files.

LeahLeah88 committed:
Задание №2.

Добавлено задание №2.

LeahLeah88 committed:
Добавлено задание №3.

Добавлено задание №3.

LeahLeah88 committed:
Добавлено задание №4.

Добавлено задание №4.

LeahLeah88 committed:
Добавлено задание №5.

Добавлено задание №5.

LeahLeah88 committed:
Добавлено задание №6.

Добавлено задание №6

tak333 commented on HW-3/HW-3.cpp:
```diff
@@ -0,0 +1,168 @@
+    std::string a;
+    std::cin >> a;
+
+    if (a == "понедельник")
```
Считывайте с консоли номер дня недели.
Реализуйте через switch-case.
Блоки case с одинаковым телом объедините в один.

LeahLeah88 replied:
Реализован ввод пользователем номера дня недели.
Реализация через switch-case.
Объединение нескольких блоков case с одинаковым телом

LeahLeah88 committed:
Update HW-3.cpp

Реализован ввод пользователем номера дня недели.
Реализация через switch-case.
Объединение нескольких блоков case с одинаковым телом

LeahLeah88 commented:
Реализован ввод пользователем номера дня недели.
Реализация через switch-case.
Объединение нескольких блоков case с одинаковым телом
